app/src/main/java/com/omkarmoghe/pokemap/models/map/LatLng.java

```diff
@@ -0,0 +1,119 @@
+package com.omkarmoghe.pokemap.models.map;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Created by chris on 7/24/2016.
+ */
+
+public class LatLng implements Parcelable {
```

Comment:
There are 2 additional functions:
1) translatePoint()
2) distance()

These things can be done in a seperate utility class with google's latlng, or the name of this class could be changed to avoid conflict